### ISSUE TITLE
Accessibility Audit: Not enough screen reader feedback when toggling sidebar  

### ISSUE THREAD

**Issue by brittnylapierre** [ProjectMirador/mirador#3883]
Description: An accessibility audit of Mirador was performed. It was found that there was not enough audio feedback for users with screen readers when hiding or showing the side navigations. 

Related: [3233](https://github.com/ProjectMirador/mirador/issues/3233) 

Supporting Comments:
- LIMITED SIGHT – Windows 10/Windows Narrator/EDGE - List of images disappeared when toggling the sidebar, impact on image unclear. 
- Blind – JAWS 2024 HOME/EDGE/WINDOWS 10 - Toggled the sidebar but had to guess its function 
- DEAF-BLIND – iPhone 13 Pro Max/iOS 17.2/Focus 14 Blue 4th gen/VoiceOver/Safari - Pointed out the lack of explicit indicators for the sidebar's expansion or collapse, with the visibility of next and prev buttons being the only discernible change. 
- LIMITED SIGHT – iPhone 14 Pro/iOS 17.2/VoiceOver/Safari - Uncertainty about the impact on the image when sidebar is toggled. 
- DEAF – BLIND – Ipad Pro 9/iPadOS 17.2/Focus 40 Blue 4th gen/VoiceOver/Safari - Noted the lack of clear indicators for the sidebar being expanded or collapsed. 
- DEAF-BLIND – Windows 11/laptop/JAWS 2020/PacMate 20 braille/Chrome - Lack of indication when the sidebar is being toggled.
